lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.ts

```diff
@@ -109,19 +99,25 @@ export class ContentMetadataComponent implements OnChanges, OnInit, OnDestroy {
 
     /** True if tags should be displayed, false otherwise */
     @Input()
-    displayTags = false;
+    displayTags = true;
 
     /** True if categories should be displayed, false otherwise */
     @Input()
-    displayCategories = false;
+    displayCategories = true;
```

Comment:
Why did we change default for displayCategories? Is not that breaking change for projects which use ADF/external customer as they will need update their code to use false now?

Reply:
Hi, @DenysVuika Can you please provide more context/reason of changing this default value of displayCategories?

Reply:
reverted to original defaults